easybuild/easyconfigs/g/glew/glew-2.2.0-GCCcore-10.2.0-egl.eb

```diff
@@ -0,0 +1,44 @@
+# Contribution from the Crick HPC team
+# uploaded by J. Sassmannshausen
+# th77embl updated to GCCcore 10.2.0
+easyblock = 'ConfigureMake'
+versionsuffix = '-egl'
+# available: -glx, -osmesa, -egl  acc. to @Micket's suggestion:
+# GLEW does support GLX (onscreen or requiring VirtualGL), EGL (technically can do both onscreen and   
+# offscreen), and OSMESA (offscreen software only).
+
+name = 'glew'
+version = '2.2.0'
+
+homepage = 'https://github.com/nigels-com/glew'
+description = """The OpenGL Extension Wrangler Library (GLEW) is a cross-platform open-source
+C/C++ extension loading library. GLEW provides efficient run-time mechanisms
+for determining which OpenGL extensions are supported on the target platform."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/nigels-com/glew/releases/download/%(name)s-%(version)s/']
+sources = ['%(name)s-%(version)s.tgz']
+checksums = ['d4fc82893cfb00109578d0a1a2337fb8ca335b3ceccf97b97e5cc7f08e4353e1']
+
+github_account = 'nigels-com'
+builddependencies = [('binutils', '2.35')]
+buildopts = 'SYSTEM=linux`echo %(versionsuffix)s|sed -e "s/-glx//g"`'
+dependencies = [
+    ('Mesa', '20.2.1'),
+    ('X11', '20201008'),
+]
+
+skipsteps = ['configure']
+
+preinstallopts = 'GLEW_PREFIX=%(installdir)s GLEW_DEST=%(installdir)s '
+install_cmd = 'make install.all'
+
+sanity_check_paths = {
+    'files': ['lib/libGLEW.a', 'lib/libGLEW.%s' % SHLIB_EXT] +
+             ['bin/glewinfo', 'bin/visualinfo'] +
+             ['include/GL/%s.h' % h for h in ['glew', 'glxew', 'wglew']],
+    'dirs': ['', ]
```

Comment:
while we're at it, ['', ] doesn't seem useful. Can just use an empty list here instead